### DISCUSSION TITLE
Glitches in 007 Goldeneye

### DISCUSSION THREAD

**Discussion by no-sauce** [mupen64plus-ae/mupen64plus-ae#983] [Q&A]
When trying to load a level to play, No matter what mission I choose It always goes to the aztek mission briefing.  Also After starting the mission, It brings me to the cradle mission with major graphic issues like walking through walls and walls not appearing properly.  This game use to work in an older version fine but now has these glitches.

**Comment by fzurita** [comment]
Do you have any cheats enabled?

**Comment by no-sauce** [comment]
Yes I had cheats enabled.  After turning them off It seams to of fixed the Issues above thanks.  Weird how with the cheats now turned off I still can use the cheats and are still active until I clean my cache and data for the app.

**Reply by fzurita** [reply]
You shouldn't have to clear data. Just choosing start instead of resume should start the game fresh.

**Comment by no-sauce** [comment]
I have tryed start/reset game but that doesn't work.  It seams the cheats are still active even when the cheats are unselected/turned off until I clear cache/data for the app.  Atleast The graphics are fixed now so its playable now.

**Reply by fzurita** [reply]
Which cheats didn't want to deactivate?

**Comment by no-sauce** [comment]
nevermind sorry my bad it is working.  One question though is with all the cheats enabled I get the issues as mentioned above so I will have too narrow it down which one is not working since I have tryed just one or two with no issues.

**Comment by no-sauce** [comment]
nevermind sorry my bad it is working.  One question though is with all the cheats enabled I get the issues as mentioned above so I will have too narrow it down to the ones that don't work.
